test/correctness/simd_op_check.cpp

```diff
@@ -192,17 +193,17 @@ struct Test {
         // Define a vectorized Func that uses the pattern.
         Func f(name);
         f(x, y) = e;
-        f.bound(x, 0, W).vectorize(x, vector_width);
+        f.bound(x, PAD, W-PAD).vectorize(x, vector_width);
```

Comment:
I tried it the the way you suggested, but I get the error "Assertion failed: base >= 0".
I made the following changes in the driver file and removed the changes from simd_op_check file:
    buf.dim[0].extent = w+PAD;
    buf.dim[1].stride = w+2*PAD;
    buf.dim[0].min = -PAD;
Is this the same thing you are suggesting?